stack/rest/src/main/java/org/apache/usergrid/rest/management/ManagementResource.java

```diff
@@ -701,6 +710,50 @@ private JsonNode getMeFromUgCentral( String extAccessToken )  throws EntityNotFo
     }
 
 
+    synchronized Client getJerseyClient() {
+
+        if ( jerseyClient == null ) {
```

Comment:
Should we put synchronized in the null block? This way everyone won't have to queue up to get the client once it's instantiated.